app/controllers/actions/AuthAction.scala

```diff
@@ -50,7 +56,11 @@ class AuthActionImpl @Inject()(override val authConnector: AuthConnector, mcc: M
 
           val cdsLoggedInUser = SignedInUser(eori.get.value, allEnrolments)
 
-          block(AuthenticatedRequest(request, cdsLoggedInUser))
+          if (eoriWhitelist.allow(cdsLoggedInUser)) {
```

Comment:
`allow` doesnt seem right to me, it implies it is "doing an action" of some kind.

I'd rename it to `contains` so it reads like "if eori whitelist contains CDS logged in user"